src/index.js

```diff
@@ -1,3 +1,33 @@
-import _ from "lodash";
-import "./style.css"
-console.log("Hello world")
\ No newline at end of file
+/* eslint-disable no-unused-vars */
+import _ from 'lodash'; // Unused vars disabled for lodash
+import './style.css';
+import Task from './taskClass.js';
+
+const dataBase = [new Task('Description test1', 0), new Task('Description test2', 1), new Task('Description test3', 2)];
+
+function showTasks() {
+  const container = document.getElementById('todo-container');
+  dataBase.forEach((task) => {
+    const taskElement = document.createElement('li');
+    taskElement.classList.add('todo-element');
+
+    const completed = document.createElement('input');
+    completed.type = 'checkbox';
+    taskElement.appendChild(completed);
+
+    const description = document.createElement('p');
+    description.innerText = task.description;
+    taskElement.appendChild(description);
+
+    const moveicon = document.createElement('span');
+    moveicon.classList.add('material-icons');
+    moveicon.innerText = 'more_vert';
+    taskElement.appendChild(moveicon);
+
+    container.appendChild(taskElement);
+  });
+}
+
```

Comment:
- [ ] It would be better if the tasks should appear in order of the `index` values for each task. This is the function of the index for every task. Currently, the tasks are rendering in order as they are in the array. kindly fix this. 
🛠️  🥇

Reply:
Properly addressed